pyproject.toml

```diff
@@ -26,6 +26,28 @@ classifiers = [
 urls = { Homepage = "https://github.com/Marker-Inc-Korea/AutoRAG" }
 dynamic = ["version", "dependencies"]
 
+[tool.poetry]
+name = "AutoRAG"
+version = "v0.1.0"
```

Comment:
Please use the dynamic version. We load our version form the `VERSION` file.

Reply:
It seems that tool.poetry does not directly support dynamic versioning, and since the version attribute is mandatory, it is recommended to specify a base version. 

The base version mentioned here appears to refer to the very first version. Therefore, I confirmed that the initial release is 0.2.0 and set it accordingly. 

Refer to this for more information: https://python-poetry.org/docs/main/pyproject/#version